### PR TITLE
snapshot: some improvments to the snapshot process

### PR DESCRIPTION
**Pull request by huikang**
### Description

snapshot: some improvments to the snapshot process

- Add `raft` to the snapshot logger because the log messages are from raft package, making it consistent with other snapshot logs like `starting snapshot up to`, which are from the raft package as well. Also it allows users know where to search for the logger message

```
// before
2023-05-08T10:10:19.954-0400 [INFO]  agent.server.raft: starting snapshot up to: index=30
2023-05-08T10:10:19.954-0400 [INFO]  agent.server.snapshot: creating new snapshot: path=/tmp/dc-2-consul-server-1/raft/snapshots/3-30-1683555019954.tmp
2023-05-08T10:10:19.967-0400 [INFO]  agent.server.raft: snapshot complete up to: index=30

// after
2023-05-08T10:04:20.361-0400 [INFO]  agent.server.raft: starting snapshot up to: index=25
2023-05-08T10:04:20.361-0400 [INFO]  agent.server.raft.snapshot: creating new snapshot: path=/tmp/dc-2-consul-server-1/raft/snapshots/2-25-1683554660361.tmp
2023-05-08T10:04:20.382-0400 [INFO]  agent.server.raft: snapshot complete up to: index=25
2023-05-08T10:04:20.382-0400 [INFO]  agent.server: creating temporary file of snapshot: path=/var/folders/c0/0_4qftyd47g8bkq_4_5dpw4m0000gn/T/snapshot3125735996
```

- Print out the path of the temp snapshot file for trouble shooting

- Update the doc a bit

### PR Checklist

* [ ] updated test coverage
* [x] external facing docs updated
* [ ] appropriate backport labels added
* [x] not a security concern

### CODE REVIEW

**Review by trujillo-adam — APPROVED** [review]
Just a few suggestions for the docs. Approving so you're not blocked on my end.

File: website/content/commands/snapshot/save.mdx
Comment (huikang) [reply website/content/commands/snapshot/save.mdx:33]: Yes, this is a result from the raft library.

Comment (huikang) [reply website/content/commands/snapshot/save.mdx:33]: But I'd like to have an eng reviewer to confirm. Thanks, @trujillo-adam

**Review by kisunji — APPROVED** [review]

**Pull request merged**
